app/assets/v2/js/pages/bounty_details.js

```diff
@@ -271,14 +347,15 @@ var pendingChangesWarning = function(issueURL, last_modified_time_remote, now){
 
 window.addEventListener('load', function() {
     setTimeout(function(){
-        var issueURL = getParam('url');
+        var issueURL = getParam('url');   // What is this url coming from?
```

Comment:
it's the URL portion of this link:

https://stage.gitcoin.co/funding/details?url=https://github.com/gitcoinco/web/issues/201

Reply:
Yup, I will remove that comment.